cores/reset.c

```diff
@@ -26,7 +26,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.   
  */
 #include <reset.h>
-#ifdef XMC1100_Boot_Kit
+#if defined(XMC1100_Boot_Kit) || defined(XMC1400_Arduino_Kit)
```

Comment:
Change to board agnostic method here using define in pins_arduino.h (also on XMC1100 Boot Kit)

#define HAS_GPIO_RESET

so
#ifdef HAS_GPIO_RESET

Reply:
Same as before when you change from ONE to more you put in a better solution AT THAT point